src/renderer/actions/paths.js

```diff
@@ -5,23 +5,99 @@ import {remote} from "electron";
 export const platforms = {stable: "Discord", ptb: "Discord PTB", canary: "Discord Canary"};
 export const locations = {stable: "", ptb: "", canary: ""};
 
-const getDiscordPath = function(releaseChannel) {
+
+/* Checks which paths exist. */
```

Comment:
Make these kind of comments jsdoc compliant:

```js
/** Checks which paths exist. */
```

Reply:
Gotcha. I am not a primary Javascript Developer (I do mostly Rust and C/C++). So I wasn't aware of that convention. Will do!

Reply:
Merely fyi: It's more than a convention, it tells IDEs that this comment describes the function and integrates with intelli-sense. You can even provide type information and descriptions for parameters and return values.
It's also used to extract/generate documentation.